src/Tools/Shared/SecretsHelpers/ProjectIdResolver.cs

```diff
@@ -59,9 +59,9 @@ public string Resolve(string project, string configuration)
                 UseShellExecute = false,
                 ArgumentList =
                     {
-                        "msbuild",
+                        "build",
```

Comment:
Are there any gotchas we should be on the lookout for as we move from `msbuild` to `build` for this? Also, I assume this exists because there are file-based app behaviors that are not available in `msbuild`?

Reply:
msbuild is the 'raw' interface. it doesn't do multi-processing by default, doesn't do terminal logger (though you're not going to see that one), it doesn't do implicit restores, etc. build will do all of those (and it has the nice CLI args for some of the common properties) in addition to supporting the file-based apps. I'd say in general you should expect more churn in the _behavior_ of build, but we do try to keep the contract pretty clean and consistent.